examples/celery-rabbitmq/README.md

```diff
@@ -31,28 +31,29 @@ You should already have turned up a Kubernetes cluster. To get the most of this
 
 The Celery task queue will need to communicate with the RabbitMQ broker. RabbitMQ will eventually appear on a separate pod, but since pods are ephemeral we need a service that can transparently route requests to RabbitMQ.
 
-Use the file `examples/celery-rabbitmq/rabbitmq-service.json`:
-
-```js
-{
-  "id": "rabbitmq-service",
-  "kind": "Service",
-  "apiVersion": "v1beta1",
-  "port": 5672,
-  "containerPort": 5672,
-  "selector": {
-    "app": "taskQueue"
-  },
-  "labels": {
-    "name": "rabbitmq-service"
-  }
-}
+Use the file `examples/celery-rabbitmq/rabbitmq-service.yaml`:
+
+```yaml
+apiVersion: v1beta3
+kind: Service
+metadata:
+  labels:
+    name: rabbitmq-service
```

Comment:
Any reason why this label value is different than those on the controllers and pods?